cmd/lncli/commands.go

```diff
@@ -2671,7 +2671,7 @@ var queryRoutesCommand = cli.Command{
 		},
 		cli.Int64Flag{
 			Name:  "num_max_routes",
-			Usage: "the max number of routes to be returned (default: 10)",
```

Comment:
What's the rationale for removing this note?

Reply:
It doesn't need to be documented there because it is defined just below in the `value` param, which automatically gets appended to the cli help text output. See my original comments on the issue that shows the output before and after this change.

```
		cli.Int64Flag{
			Name:  "num_max_routes",
			Usage: "the max number of routes to be returned",
			Value: 10, <----- this
		},
```